src/analysis/asymptotics/asymptotics.lean

```diff
@@ -467,6 +467,21 @@ is_o.of_is_O_with $ λ c cpos, (h.forall_is_O_with cpos).sup (h'.forall_is_O_wit
 @[simp] lemma is_o_sup : f =o[l ⊔ l'] g ↔ f =o[l] g ∧ f =o[l'] g :=
 ⟨λ h, ⟨h.mono le_sup_left, h.mono le_sup_right⟩, λ h, h.1.sup h.2⟩
 
+lemma is_O_with_insert [topological_space α] {x : α} {s : set α} {C : ℝ} {g : α → E} {g' : α → F}
```

Comment:
```suggestion
lemma is_O_with_insert_iff [topological_space α] {x : α} {s : set α} {C : ℝ} {g : α → E} {g' : α → F}
```
(Otherwise I would expect the lemma to give only the right to left direction, which is the nontrivial one). Same thing for several other lemmas. With an alias, you could also register `is_O_with.insert` for the useful direction, in this lemma and the other ones.

Reply:
Are you happy with `is_O_with_insert` for the `iff` and `is_O_with.insert` for the nontrivial direction? I think that is a quite common naming pattern in the library.

Reply:
I added the useful direction explicitly. I think the lemmas in the other files already have aliases.